### ISSUE TITLE
Camera Sleep and Timeout fetching status / ezviz_cloud data

### ISSUE THREAD

**Issue by mindtripper** [RenierM26/ha-ezviz#8]
I've noticed that the component does not update status and in cant fetch data when the camera is in Sleep mode.
`2021-01-11 10:25:51 ERROR (MainThread) [custom_components.ezviz_cloud.coordinator] Timeout fetching ezviz_cloud data`
Using py-ezviz to fetch status when the camera is in sleep mode takes about 15 seconds.
```
bash-5.0# date && pyezviz .... status && date
Mon Jan 11 10:32:25 CET 2021
{
  "serial": "..........",
  "name": "C6N(..........)",
  .......
  "sleep": true,
  "detection_sensibility": "Unknown",
  .......
}
Mon Jan 11 10:32:41 CET 2021
```
When the camera is online, the time to fetch status is about 2 seconds.
bash-5.0# date && pyezviz .... status && date
Mon Jan 11 10:34:04 CET 2021
```
{
  "serial": "..........",
  "name": "C6N(..........)",
  .......
  "sleep": false,
  "detection_sensibility": "2",
  .......
}
Mon Jan 11 10:34:06 CET 2021
```

Is it related to how slow the fetching of data is that is causing this behavior? Does HA component timeout?

Note: I prefer sleep mode compared to privacy mode when I want complete privacy.

**Comment by mindtripper** [comment]
Update:

Increasing timeout value in https://github.com/RenierM26/ha-ezviz/blob/main/custom_components/ezviz_cloud/coordinator.py#L38 to 45 seconds solves this issue. The camera is detected and status updated. `sleep: true`.

**Comment by mindtripper** [comment]
I have reduced the timeout to 35 seconds. Still detected when the camera is sleeping.

Suggestion:
Implement restore state in the component. Checking if the camera was sleeping last time HA was running. If it was. Allow longer timeout fetching camera state.
[Restoring An Entity on Home Assistant Restart](https://aarongodfrey.dev/programming/restoring-an-entity-in-home-assistant/)

**Comment by RenierM26** [comment]
Hi mindtripper,

I have increased the coordinator timeout (35 seconds) in the latest release (0.0.2.5) and also fixed the timeout option setting. The later will allow timeouts on the https requests on the api. (you might also need to set this but the default of 10 seconds per https request should be fine.)

Thanks for the link to "restore state"! I've also implemented this in the latest release and removed the "force update" on startup flag on initial setup.
